src/ipyforcegraph/behaviors/wrappers.py

```diff
@@ -32,6 +41,10 @@ class WrapperBase(ForceBase):
         help="the ``DynamicValue``, ``WrapperBase``, or string wrapped by this wrapper",
     ).tag(sync=True, **W.widget_serialization)
 
+    def __init__(self, wrapped: TAnyWrapped, **kwargs: Any):
+        kwargs["wrapped"] = wrapped
+        super().__init__(**kwargs)
+
```

Comment:
at present, this would conflict with `Tint` which uses `value` directly... let's think of a more specific name, and see if `traitlets` can do the heavy lifting for us...

Reply:
I think adding the `get` form (but as a trait) is fairly cut-and-dry. However, making it settable introduces the chance of creating cycles, which is probably not worth the bookkeeping to avoid.

Will push forward with the less-conflicty `objective`... `object` would be better, but overloading builtins and all that jazz...

Reply:
https://github.com/jupyrdf/ipyforcegraph/issues/132